src/main/java/com/raeic/embarker/party/commands/JoinCommand.java

```diff
@@ -108,6 +109,9 @@ public boolean onCommand(CommandSender sender, Command command, String label, St
                 PartyPlayer newPartyPlayer = new PartyPlayer(playerUniqueId, inviteSenderPlayerUniqueId);
                 newPartyPlayer.save();
 
+                // Invalidate the existing party as we updated it.
+                Globals.party.invalidateCacheByKey(inviteSenderPlayerUniqueId);
```

Comment:
What if we moved this into PartyPlayer#save? Theoretically whenever a PartyPlayer is saved, the cache will be invalidated, right?

Reply:
thats true is a good idea